boards/arm/stm32f429i_disc1/stm32f429i_disc1_defconfig

```diff
@@ -23,6 +23,12 @@ CONFIG_PINMUX=y
 # enable GPIO
 CONFIG_GPIO=y
 
+# enable FMC
+CONFIG_MEMC=y
+
+# enable SDRAM
+CONFIG_MEMC_STM32_SDRAM=y
```

Comment:
As discussed, board should not enable any of the `MEMC` options. Test/sample takes care of that.